plugins/main/host-device/host-device.go

```diff
@@ -625,3 +633,33 @@ func validateCniContainerInterface(intf current.Interface) error {
 
 	return nil
 }
+
+func cmdStatus(args *skel.CmdArgs) error {
```

Comment:
We may check the host-device's existence for status. How about to do that (or add comments for TODO?)